independent-publisher-connectors/Exact Online Time & Billing/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
```

Comment:
@Knots23 I hope you are doing well.

Please accept my apologies for long delay in the progress. We would like to suggest a new approach that will require users to provide their own credentials for the connector. please see examples below.
[ebay](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/independent-publisher-connectors/eBay/apiProperties.json#:~:text=%22properties%22%3A%20%7B-,%22connectionParameters%22%3A%20%7B,%7D%2C,-%22iconBrandColor%22%3A)
[Quickbooks](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/independent-publisher-connectors/TSheets%20Quickbooks%20Time/apiProperties.json)

If you would like to change the apiProperties to support the above approch, we would be happy to publish the connector quickly for you.
I look forward to hearing from you. 

Thank you so much again.